src/lingrex/trimming.py

```diff
@@ -1,201 +1,182 @@
 """
-Trimming functionalities in lingrex.
+Functionality to trim alignments by removing sites.
 """
 import random
+import typing
+import functools
+import itertools
 import collections
-from lingpy.sequence.sound_classes import token2class
-
 
-def revert(alms):
-    return [[row[i] for row in alms] for i in range(len(alms[0]))]        
+from lingpy.sequence.sound_classes import token2class
 
+from lingrex.util import subsequence_of
 
-def get_skeleton(alms, gap="-"):
-    return [token2class(([c for c in col if c != gap] or ["+"])[0], "cv") for col in revert(alms)]
+__all__ = ['GAP', 'Site', 'Sites', 'prep_alignments']
```

Comment:
I moved `prep_alignments` here because having a function in `util` that imports from one of the non-util modules feels wrong.